packages/astroport-governance/src/builder_unlock.rs

```diff
@@ -18,13 +18,16 @@ pub struct State {
     pub total_astro_deposited: Uint128,
     /// Currently available ASTRO tokens that still need to be unlocked and/or withdrawn
     pub remaining_astro_tokens: Uint128,
+    /// Amount of ASTRO tokens deposited into contract but without allocation
+    pub unallocated_tokens: Uint128,
 }
 
 impl Default for State {
     fn default() -> Self {
         State {
             total_astro_deposited: Uint128::zero(),
             remaining_astro_tokens: Uint128::zero(),
+            unallocated_tokens: Uint128::zero(),
```

Comment:
These are defaults for Uint128. I suggest to remove this impl block and use ```#[derive(..., Default)]``` for State struct.

Reply:
Done